examples/gamepad.html

```diff
@@ -0,0 +1,162 @@
+<!--
+  Copyright 2016 Google Inc. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>A Simple Demo: Omnitone with Headtracking</title>
+  <script src="../build/omnitone.min.js"></script>
+  <style>
+    h2 {
+      font-family: sans-serif; 
+    }
+
+    button {
+      width: 120px;
+      height: 24px;
+      font-weight: bold;
+    }
+
+    select {
+      width: 120px;
+      font-size: 24px;
+    }
+  </style>
+</head>
+<body>
+  <h2>Select the sound source to play.</h2>
```

Comment:
We might want to add a header with `<h1>` to say this demo is about the gamepad.